app/models/permissions/scientist_permission.rb

```diff
@@ -1,19 +1,18 @@
 # frozen_string_literal: true
 
 ##
-# Permissions for a Standard User
-# Allowed to create a scan in the user interface and the api.
+# Permissions for a Scientist
+#
 module Permissions
   class ScientistPermission < BasePermission
     def initialize(user)
       super
       allow :scans, [:create]
+      allow :upload_labware, [:create]
```

Comment:
I am wondering who is doing labware uploading? It may well be a techincal action. We can leave it for now though.

Reply:
ill leave a comment in the code